admin/packaging.py

```diff
@@ -662,6 +662,16 @@ def __str__(self):
 
         # Cryptography hazmat bindings
         'package-installs-python-pycache-dir opt/flocker/lib/python2.7/site-packages/cryptography/hazmat/bindings/__pycache__/',  # noqa
+
```

Comment:
The first does seem OK.

The second is a bit more complicated, and I don't know the full answer. I'm pretty sure that in a lot of cases we assume that the ubuntu distro implies "not systemd". I think the best way to find out is by building this test image and (somehow!) running the relevant tests against it. I don't know the exact details of how to do this, but will offer what help I can. 

I think this is worth doing soon, even if we don't make any changes to actual systems until next week.